Core/UIViewControllerExtension.swift

```diff
@@ -27,7 +27,7 @@ extension UIViewController {
     }
     
     var isPad: Bool {
-        return traitCollection.horizontalSizeClass == .regular
+        return UIDevice.current.userInterfaceIdiom == .pad
```

Comment:
This would cause a problem on iPad when it launched as the "slim" view in Split View mode.  I tested the other thing this affects and it seems to work as expected.